core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/util/IObservable.java

```diff
@@ -30,7 +30,7 @@
  * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
  * WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.openjdk.jmc.ui.common.util;
+package org.openjdk.jmc.common.util;
 
 import java.util.Observable;
 import java.util.Observer;
```

Comment:
**thought:** Should this IObservable be even removed, both `Observer` and `Observable` are deprecated since JDK 9.

Reply:
Good point, think that might deserve it's own issue in JIRA though